docs/ext/http.rst

```diff
@@ -109,3 +109,10 @@ See :ref:`config` for general help on configuring Mopidy.
 
     This config should only be disabled if you understand the security implications
     and require the HTTP server's old behaviour.
+
+
+.. confval:: http/default_webclient
```

Comment:
I think I prefer `http/default_app`. We're already in the context of http, so web is given. Also, the app might be something else than a client, like the API Explorer web extension.

@adamcik @kingosticks Any thoughts? The name is the only thing we cannot easily change later.

Reply:
Well, I think I used default_webapp at some point, that we change, since the http serves both "apps" and "statics" files, so, naming the conf element "app" is sort of incomplete for me. (and yes, "web" is given)

Reply:
well, early versions was with a "app" something name, but it excludes static served pages.

Reply:
The names "apps" and "statics" in the HTTP code is mostly an internal thing. They could just as well have been named "dynamic_apps" and "static_apps". I think most people think of a pure client-side client as an app.

(Sidenote: I kind of hope not too many are using the dynamic apps, so that we easier can switch away from Tornado if we want to do that in the future.)

Reply:
App to me seems a bit weird and we refer to the as 'webclients' in other places (mopidy.com/ext for one) but I'm not too bothered. I think either works and is pretty obvious.

Reply:
If we rate them as equivalent, I'd say "default_app" wins on being a lot shorter.

Reply:
Done